### ISSUE TITLE
Forgot about this - Combat Level Formula Attribute is busted

### ISSUE THREAD

**Issue by RVNJ** [phoenixanimations/TheNuwasFile#77]
always only ever displays 1

**Comment by phoenixanimations** [comment]
Can you give step by step on how you got the bug, as if you go into: ObjectsTest->XNodeTest->Combat Level Test it still works.

Did you not have it set up with a start and end node, the combat formula only works when you hit "run". Unlike character sheet.

**Comment by RVNJ** [comment]
Yeah, it has all of the connections. I'll get back to you on the step-by-step reproduction, it was just such an odd occurrence because it was so isolated.

To do:
- [ ] Reproduce CBLF LV 1 Bug

**Comment by RVNJ** [comment]
Yeah, it's still bugged for me, not sure why.

Have you set both the Combat Level stat and Melee/Magic/Archery (not Current and not Experience) Stats to values that are obscure?

> Are you testing this on your side or mine? Try testing it on Arcanym if you're doing the tests on your side. I just tried everything I could and I'm reproducing the bug 100% of the time, so I have no clue what to do to fix it.
